web/src/components/profile/ProfileOpportunitiesAlert.tsx

```diff
@@ -1,11 +1,10 @@
-import { Content } from "@/components/Content";
-import { Alert } from "@/components/njwds-extended/Alert";
 import { ProfileDataContext } from "@/contexts/profileDataContext";
 import { useConfig } from "@/lib/data-hooks/useConfig";
 import { getProfileConfig } from "@/lib/domain-logic/getProfileConfig";
 import { ProfileContentField } from "@/lib/types/types";
 import { getFieldsForProfile, isFieldAnswered } from "@businessnjgovnavigator/shared";
 import { ReactElement, useContext } from "react";
+import { FieldEntryAlert } from "../FieldEntryAlert";
 
 export const ProfileOpportunitiesAlert = (): ReactElement => {
```

Comment:
[dust] I think this component is unnecessary because there is not a lot of logic here. What are your thoughts on deleting it?

Reply:
Completely misread this.

We use this component 4 times in `profile.tsx`. From an organization perspective, this makes sense to leave it as a separate component.

A week or so ago we were talking about when logic should be extracted from parent into a dedicated component. I think this is a good case. `profile.tsx` is 807 lines. Keeping these 40 lines in their own file creates a separation of logic.